config/rules.rb

```diff
@@ -376,8 +376,9 @@ def summarise_all_feedback(summaries, opp)
     if (opp['tags'] & ['Software', 'Hardware']).any?
       value = opp['tags'].include?('Software') ? 'software' : 'hardware'
       # set value for most feedback form if not already present
+    
       if result[:has_app_review]
-        result[:app_review_software_hardware] ||= value
+        result[:app_review_software_hardware] = value
```

Comment:
this change will only affect opportunities with the tag "software" or "hardware" manually added, and mean the presence of one of these tags will override any value from the feedback form, even for the new forms with the soft/hardware question present. 

it will not remove the "unknown" tag for older feedback forms without the soft/hardware question where one of the manual tags has not been added to the opportunity.

is all of the above what we want?

Reply:
So we asked the teams to add the 'Software' or 'Hardware' tag for those individuals who would have been missed out because of historical app reviews - so the above should cover everyone. 

But now I'm wondering whether we should check for all unknown ones too i.e. if software/hardware is unknown then check other feedback form answers